### ISSUE TITLE
Which version of the image encoder is used in stable video diffusion?

### ISSUE THREAD

**Issue by pixeli99** [huggingface/diffusers#6648]
I'd like to know which version of the image encoder is used in stable video diffusion.

**Comment by a-r-r-o-w** [comment]
I'm not sure if this is what you mean but the CLIP image encoder is the CLIPVisionModelWithProjection from huggingface/transformers. You can find the initialization config that it uses in the [config.json](https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt/blob/main/image_encoder/config.json) file.

**Comment by pixeli99** [comment]
Yes, I understand what you mean, but I would like to know to which weight this corresponds in huggingface/transformers. In other words, I am actually looking to find the weight of the text encoder that corresponds to this image encoder.
